src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return super.toString();
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }

```

Comment:
Where is overridden `toString()`?
![Screenshot 2022-06-24 at 08 06 30](https://user-images.githubusercontent.com/60258373/175466213-21fe41be-fc1d-4e3a-bb21-255d0fce2a0f.png)